app/src/main/java/org/oppia/android/app/profile/ProfileInputView.kt

```diff
@@ -1,131 +0,0 @@
-package org.oppia.android.app.profile
-
-import android.content.Context
-import android.text.InputFilter
-import android.text.InputType
-import android.text.TextWatcher
-import android.util.AttributeSet
-import android.view.LayoutInflater
-import android.view.View
-import android.widget.EditText
-import android.widget.LinearLayout
-import android.widget.TextView
-import androidx.databinding.BindingAdapter
-import org.oppia.android.R
-import org.oppia.android.app.application.ApplicationInjectorProvider
-import org.oppia.android.app.shim.ViewBindingShim
-import javax.inject.Inject
-
-/** Custom view that is used for name or pin input with error messages. */
-class ProfileInputView @JvmOverloads constructor(
```

Comment:
@rt4914 We are removing `ProfileInputView` completely in this pr. Let me know if you have anything to add here.

Reply:
@anandwana001 I thinks that's great. Lets remove it completely.